src/lib/components/BucketAggregation/BucketAggregationValues.js

```diff
@@ -127,7 +138,7 @@ const ValueElement = (props) => {
     >
       <List.Item key={bucket.key}>
         <Checkbox
-          label={label}
+          label={label ? label : bucket.key}
```

Comment:
I think this check is redundant as you check the label before no?

Reply:
👍